.travis/install-ivykis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x -e
+
+OLDPWD=`pwd`
+
+cd $TRAVIS_BUILD_DIR
+curl -sL https://github.com/balabit/ivykis/archive/master.zip -o ivykis.zip
```

Comment:
I'd implement this when we receive the first failure. "A Good Plan Today Is Better Than a Perfect Plan Tomorrow" :)